multiwallet.go

```diff
@@ -56,80 +52,62 @@ func NewMultiWallet(rootDir, dbDriver, netType, politeiaHost string) (*MultiWall
 		return nil, err
 	}
 
-	rootDir = filepath.Join(rootDir, netType)
-	err = os.MkdirAll(rootDir, os.ModePerm)
-	if err != nil {
-		return nil, errors.Errorf("failed to create rootDir: %v", err)
-	}
-
-	err = initLogRotator(filepath.Join(rootDir, logFileName))
-	if err != nil {
-		return nil, errors.Errorf("failed to init logRotator: %v", err.Error())
-	}
-
-	mwDB, err := storm.Open(filepath.Join(rootDir, walletsDbName))
-	if err != nil {
-		log.Errorf("Error opening wallets database: %s", err.Error())
-		if err == bolt.ErrTimeout {
-			// timeout error occurs if storm fails to acquire a lock on the database file
-			return nil, errors.E(ErrWalletDatabaseInUse)
-		}
-		return nil, errors.Errorf("error opening wallets database: %s", err.Error())
-	}
-
-	// init database for saving/reading wallet objects
-	err = mwDB.Init(&Wallet{})
-	if err != nil {
-		log.Errorf("Error initializing wallets database: %s", err.Error())
-		return nil, err
-	}
-
-	// init database for saving/reading proposal objects
-	err = mwDB.Init(&Proposal{})
+	dcrDB, dcrRootDir, err := initializeDCRWallet(rootDir, dbDriver, netType)
```

Comment:
Related to my first comment, this db isn't a dcrDB; it's a multiwallet db and should use the rootDir without appending `"dcr"`. This db would track all wallets for all assets but the individual wallet dbs would be in the asset subfolders.